src/index.ts

```diff
@@ -0,0 +1 @@
+export const getGreeting = () => 'Welcome To PantryNode!!';

```

Comment:
Code is a liability. I don't think this code is needed in the product and therefore, we shouldn't add unnecessary code.

Reply:
Moreover, if that line was deleted in the future, it would then cause the smoke test to fail. That means the smoke test would be indicating that the testing infrastructure didn't work (even though it appears it does).

The simple solution is to make the smoke tests *self-reliant*. They shouldn't invoke any other production code, they should simply verify an unfalsifiable assertion (`expect`) like making sure `true` is `true` or that `1+1` is equal to `2`

Reply:
Removed index.ts file and added 1+1 is equal to 2 in smoke test - 
<img width="526" alt="image" src="https://user-images.githubusercontent.com/22402315/229726086-11839cba-6ed6-4d68-b9b5-d3b3a4ad4818.png">